content/news/release-20151224.md

```diff
@@ -0,0 +1,61 @@
+40 developer elves have toiled long and hard to fill your stocking with lots of fixes, improvements and long-awaited features.
+
+In particular, this release re-introduces the global chat channel that is shared across all mods, which will allow you to meet up with other waiting players and arrange matches without joining servers. Going along with it is a re-design of the ingame server browser as well as the skirmish lobby.
+
+Also new in this release is a game speed selector, a feature that many people have often and repeatedly asked for over the years, and now it's finally here, for multiplayer, single-player skirmish and missions!
+
+In addition, OpenRA now offers an additional way of scrolling the map similar to the way it worked in the original Tiberian Sun and Red Alert 2 (by right-clicking and moving the mouse). To use it, select the "Joystick" scrolling option in the input settings dialog.
+
+A change we are sure many will rejoice to hear is that units will now start with the "Defend" stance by default. No more losing units that decided to hunt after enemies without your explicit orders!
+
+Another big focus of this release has been overhauling our D2K mod.  We have added Thumpers, spice blows, and rebalanced all unit and 
+building statistics to more closely match the original game.
+
+The Red Alert mod has received some much-needed balance changes, with a focus on making the country-specific units more interesting and useful. The changes are too numerous to list them all here, have a look at [the changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) for all the details. There are also a total of five new missions for the Soviet single-player campaign, which means that it is the first of our campaigns to be 50% complete!
+
+The Tiberian Dawn mod also includes two new single-player missions, as well as some minor balance changes and several new multi-player maps.
+
+As usual, we not only have treats for our players, but for modders as well. The map editor has seen further improvements, such as a cash counter and a copy and paste function. There are also new traits for weather and lighting effects, more flexible `Missile` and `DeployToUpgrade` logic, as well as new additions to the Lua API. As usual, use the OpenRA.Utility to upgrade your mods or maps to this new release. Be warned, however, that there are couple of changes this time around where it unfortunately wasn't possible to offer an automated upgrade path. Most of these are mentioned in the changelog, the OpenRA.Utility will alert you to the remaining ones when it runs.
+
+There are a lot more minor changes and bugfixes, so take a look at the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) if you want to unwrap all of the details.
+
+Grab the installer now for your operating system from [our download page](/download/)! We hope you'll enjoy this newest installment of OpenRA!
+
+The OpenRA development team wishes you a Merry Christmas and a Happy New Year!
```

Comment:
Could we move this onto the banner at the top (either as a static overlay, or using a text popup ingame after the crates land)?  It would add helpful context to the purpose of it, as right now it doesn't have a conclusion, and isn't referenced anywhere in the post.